cmd/traffic_layout/traffic_layout.cc

```diff
@@ -153,14 +153,14 @@ produce_layout(bool json)
   if (json) {
     printf("{\n");
   }
-  print_var("PREFIX", Layout::get()->prefix, json, false); // Don't free this
+  print_var("PREFIX", ats_strdup(Layout::get()->prefix.c_str()), json, false); // Don't free this
```

Comment:
print_var does not take const char*. I will change the code in print_var() and remove the ats_free part to get rid of this ats_strdup.